### PR TITLE
Add empty states

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: views/partials/emptyModule.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class='module'>
+    <h1>{{ 'noContentLabel'|trans(current_locale.code) }}</h1>
```
Comment (Roblinde) [inline views/partials/emptyModule.html]: This should be a h2 / h3 not a h1 / h2 I believe.

Comment (dlitvakb) [reply views/partials/emptyModule.html]: I stole this from the JS implementation.

Not sure what the proper value should be then.

Comment (dlitvakb) [reply views/partials/emptyModule.html]: Looks like JS updated it a few days ago, and I had the older template.

Updating

Comment (Roblinde) [reply views/partials/emptyModule.html]: It should be h2, h3. I'll paste you the slack message deciding this.

**Pull request merged**